docs/csharp/includes/csharp-version-alert.md

```diff
@@ -0,0 +1,9 @@
+---
+author: IEvangelist
+ms.author: dapine
+ms.topic: include
+ms.date: 09/08/2020
+---
+
+> [!IMPORTANT]
+> C# is versioned. Depending on the version of C# you're using, various features may or may not be available. C# versions are *defaulted* based on the target framework. For more information, see [C# language versioning defaults](../language-reference/configure-language-version.md#defaults).
```

Comment:
I think it's important to guide readers to the version we document. How about something like this instead of the first sentence:

```suggestion
> The official documentation tracks the latest C# version. We are currently writing for C# 9.0. Depending on the version of C# you're using, various features may not be available. The *default* C# version for your project is based on the target framework. For more information, see [C# language versioning defaults](../language-reference/configure-language-version.md#defaults).
```